### ISSUE TITLE
click.discord.com: breakage

### ISSUE THREAD

**Issue by Maxhem2** [uBlockOrigin/uAssets#18516]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Is blocked by [Peter Lowe’s Ad and tracking server list](chrome-extension://cjpalhdlnbpafiamejdnhcphjbkeiagm/asset-viewer.html?url=plowe-0)

That adress is a normal discord website and used to for example verify discord location.

### URL(s) where the issue occurs.

`click.discord.com`

### uBO version

newest

### Browser name and version

chrome latest

### Settings

no

**Comment by okiehsch** [comment]
>  used to for example verify discord location

That's probably the reason it was added to that list, if it does not break anything.
@pgl

**Comment by Maxhem2** [comment]
Thats not what an adblocker is supposed to do. A user must access that site to verify discord login on a new device/location

**Comment by okiehsch** [comment]
If it breaks site functionality we will badfilter it, if it completely breaks tha ability to access the site I am sure @pgl will remove it.

Add
`||click.discord.com^$badfilter`
to your filter list for now.

**Comment by stephenhawk8054** [comment]
> Thats not what an adblocker is supposed to do

uBO is not an adblocker, it's a [wide-spectrum content blocker for Chromium and Firefox, including ads, trackers, coin miners, popups, annoying anti-blockers, malware sites...](https://github.com/gorhill/uBlock#ublock-origin-ubo) If it's considered as trackers by Peter Lowe's list, it's supposed to be blocked as expected.

**Comment by stephenhawk8054** [comment]
`click.discord.com` is discord's email click-tracking service, like any other click-tracking vendors. A normal URL in email looks like this: [https://support.discord.com/hc/articles/218410947](https://click.discord.com/ls/click?upn=qDOo8cnwIoKzt0aLL1cBeNlOcN7VC1Mue2BSa5oqYEerY7rl5RMLJoApfxCy9y4PN7TcFbo350MxcVCB2OIz-2BQ-3D-3DrYsX_eUmL5OyfJvCu6JsP4cGgaGQTiCVYfzHRbROBfea5QaqkL1nm-2BCDcrPttv2umqfCBOniIuCcf80M2yYn-2FRX1gZGyb-2BzF-2BtnRBtY5LKnj-2F0M9O3-2Fq86LxeXuBzrkRlQLRe3WlyZXBuBUmIPR0oCbZy2SrjCP5E7sKaOtj9euUo1YNih1yKLtPsw4UmbyJvozhoCYETA4qJcuP8QNnMwoE6X7Tn8k7ot0XLuTcTNsA48hhIg7noYprPGb69QvNV).

It's simply a click-tracking redirection and does not affect any other discord's functionalities. It's not used for `verify discord location`. The email's screenshot above is simply because of discord detects the login IP (a suspicious IP), not related at all to `click.discord.com`.

If we would ever need to `badfilter` it, it will just be because of its popularity.

**Comment by okiehsch** [comment]
> . The email's screenshot above is simply because of discord detecting the login IP (a suspicious IP) address, not related at all to how click.discord.com works.

So you can verify with the filter in place?
I don't use discord, so I can't test but if that's the case we can close the issue as won't fix.

**Comment by stephenhawk8054** [comment]
> So you can "verify login" with the filter in place? I don't use discord, so I can't test but if that's the case we can close the issue as won't fix.

Although I haven't had that specific email but I'm almost sure it's just an instertitial link, the real link to verify login will appear after user clicking on `Proceed` in strict-block page. I haven't had that specific email but for the similar buttons of `Verify email` when signing up and `Reset password` that I have encountered, they are just redirections to the real links.

**Comment by Maxhem2** [comment]
> My demo link:
> 
> `https://click.discord.com/ls/click?upn=qDOo8cnwIoKzt0aLL1cBeJWc43CAiLlKYSQGUErhKV7fF2lroxEuEaMS14HcVQRWA4vev6z5-2FD7pMm0UmIHHH3PVBlYd4uSMBKKcF5FMbFXRtBCp6gKsVKW9jc9py89-2Bo4JW-2F3WqkfjM-2BtwI23qVI-2FMcmPNe3xMHQGs7qWuWk-2Fu822FOF4jy4pW6odFv0YzUEmYx_lWptfmvEGZGGT72YNT0dS3TdQTKQbeVC6s9Lzg6j-2FzEVgDtf7HS2-2F8XCM7IBcL1IYvUBcdoi-2FjQ0-2BVknGNKjJuZugCUM7QX8vAI41kM9gNTU9snqnCxnAX-2FAs9pCZby1HMrfRpOVnCPR-2FL6XIg88k3dTw83DeVcTOL6h75UhEmj53T1M1-2BnBRZyOQ-2B8dnXc73mtXdKWqPoQBFcZCE5h3C3ymCDdyb6z8JTEdCM33BsMZgVkSfYJlNUG5XHe8mQvb`
> 
> And where redirect:
> 
> `https://discord.com/authorize-ip#token=WzQ1Nzk5MDk1NzgyNTEzMDQ5NywiMTg4LjE0Ni4yNTQuMTAyIl0.ZIsJIg.x_9iGoWlt4d3MPPyy29xWawwbPg`

Thanks, mine was similar

**Comment by Maxhem2** [comment]
> To clarify more, `click.discord.com` is discord's email click-tracking service, like any other click-tracking vendors. Any URL in discord emails will look like this: [https://support.discord.com/hc/articles/218410947](https://click.discord.com/ls/click?upn=qDOo8cnwIoKzt0aLL1cBeNlOcN7VC1Mue2BSa5oqYEerY7rl5RMLJoApfxCy9y4PN7TcFbo350MxcVCB2OIz-2BQ-3D-3DrYsX_eUmL5OyfJvCu6JsP4cGgaGQTiCVYfzHRbROBfea5QaqkL1nm-2BCDcrPttv2umqfCBOniIuCcf80M2yYn-2FRX1gZGyb-2BzF-2BtnRBtY5LKnj-2F0M9O3-2Fq86LxeXuBzrkRlQLRe3WlyZXBuBUmIPR0oCbZy2SrjCP5E7sKaOtj9euUo1YNih1yKLtPsw4UmbyJvozhoCYETA4qJcuP8QNnMwoE6X7Tn8k7ot0XLuTcTNsA48hhIg7noYprPGb69QvNV5CPgq8xMMQmGjHSOYdI7SMeuWA-3D-3D) (Hover over the link to see its tracking URL).
> 
> It's simply a click-tracking redirection and does not affect any other discord's functionalities nor contribute anything useful to users. It's just used for building tracking database of discord itself and totally not used for `verify discord location`. The email's screenshot above is simply because of discord detecting the login IP (a suspicious IP) address, not related at all to how `click.discord.com` works.
> 
> If we would ever need to `badfilter` it, it will just be because of its popularity.

Again, in order to login to a users discord account on a device in a new location, a user MUST click on that link to be redirected to the Ip authorization link. Please better inform yourself

**Comment by krystian3w** [comment]
Also some lists break pop-up change old #hashtag nick name to unique without #XXXX.

To reproduce I need Someone's forgotten account.

**Comment by stephenhawk8054** [comment]
> Again, in order to login to a users discord account on a device in a new location, a user MUST click on that link to be redirected to the Ip authorization link. Please better inform yourself

I know, it does not change the fact that it's discord's redirection tracker. I have given most of its information about the use of that domain.  I also said about where the link will redirect to. Which information should I give more?

It's a tracker. The one you should complain is discord. Why didn't they just give direct link to users, instead of the tracking domain?

**Comment by krystian3w** [comment]
Discord ignore lament to implement other protection like 2FA or YubiKey.

**Comment by gorhill** [comment]
> obviously aimed at mocking me.

Just a reminder that people need to be careful about what they install with minimal due diligence, this is a universal and often repeated advice regarding extensions.

**Comment by krystian3w** [comment]
Maybe his test don't allow grants access with another public IP (e.g. mobile phone IP due we have not browser and e-mail client on PC).

In addition, you need to remember about uBo lite where when strict-block occurs, we do not have the allow tmp button, but only the power button from the add-on interface: https://github.com/uBlockOrigin/uBOL-issues/issues/23#issuecomment-1368889536

**Comment by Maxhem2** [comment]
wow

**Comment by krystian3w** [comment]
These can be broken by EL: https://github.com/uBlockOrigin/uAssets/issues/18516#issuecomment-1593102150 why you colse too fast.

**Comment by MarkJeronimus** [comment]
Fixing this would cause people to be tracked unknowingly, so I understand why this is unwanted. The only remaining problem is the lack of a 'temp allow' button in uBo lite

**Comment by Yuki2718** [comment]
Duplicate of https://github.com/uBlockOrigin/uAssets/issues/17061
